src/components/MainMenu/MainMenu.js

```diff
@@ -89,35 +88,52 @@ class MainMenu extends React.Component<Props, State> {
   }
 
   setupFocusTrap() {
-    if (this.homeLink && this.addPlaceLink) {
-      this.homeLink.addEventListener('keydown', this.focusToLastElement);
-      this.addPlaceLink.addEventListener('keydown', this.focusToFirstElement);
```

Comment:
This and the other stuff below just fixes several type errors that popped up in this file.